python/pyspark/pandas/generic.py

```diff
@@ -3259,6 +3260,10 @@ def interpolate(
             Maximum number of consecutive NaNs to fill. Must be greater than
             0.
 
+        limit_direction : str, default None
```

Comment:
How about adding `versionadded`?

Reply:
the `interploate`'s `versionadded` is here https://github.com/apache/spark/pull/36246/files#diff-cb01df978681ad2406bdd22c29f73aa3020cfcd5c1f6e4052b4bc33f59160d33R3271, I think no need to add it for this param.

Reply:
My bad I didn't notice that.